Gemfile

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in eyaml.gemspec
+gemspec
+
+gem "ffi", github: "cheddar-me/ffi", branch: "apple-m1", submodules: true
+gem "rbnacl", github: "cheddar-me/rbnacl", branch: "apple-m1", submodules: true
```

Comment:
I looked at your change for `rbnacl`, but it seems to me it's not actually needed? The ffi git version should be enough